packages/core/content-releases/server/src/bootstrap.ts

```diff
@@ -52,10 +52,10 @@ export const bootstrap = async ({ strapi }: { strapi: Core.Strapi }) => {
        * so we need to fetch them before deleting the entries to save the ids on our state
        */
       async beforeDeleteMany(event) {
-        const { model, params } = event;
+        const model = strapi.getModel(event.model.uid as any);
```

Comment:
Is typecasting as the expected type better than any 🤔 ? Maybe this should be as UID.Schema or whatever it is. Also wondering if this type is accurate or if it itself should be using UID.Schema